blueoil/utils/config.py

```diff
@@ -133,19 +133,19 @@ def _load_py(config_file):
         exec(source, globals(), config)
 
     # use only upper key.
-    return EasyDict({
+    return SmartDict({
         key: value
         for key, value in config.items()
         if key.isupper()
     })
 
 
 def _easy_dict_to_dict(config):
```

Comment:
How about changing the function name?
```suggestion
def _smart_dict_to_dict(config):
```

Reply:
@oatawa1 Thanks. Changed.